java/ql/lib/semmle/code/java/security/SensitiveApi.qll

```diff
@@ -73,8 +66,4 @@ deprecated predicate javaApiCallableCryptoKeyParam(Callable c, int i) {
  * DEPRECATED: Use the `CredentialsSinkNode` class instead.
  * Holds if callable `c` from a known API expects a credential parameter at index `i`.
  */
-deprecated predicate otherApiCallableCredentialParam(Callable c, int i) {
-  exists(InitializationVectorSink sink, MethodAccess ma |
-    sink.asExpr() = ma.getArgument(i) and c = ma.getCallee()
-  )
-}
+deprecated predicate otherApiCallableCredentialParam(Callable c, int i) { none() }
```

Comment:
Hm, this breaks backwards compatibility. Maybe we need to keep those two sinks here (and only here) to avoid potentially breaking users' queries, even if we remove them from our own hardcoded credentials query:

```suggestion
deprecated predicate otherApiCallableCredentialParam(Callable c, int i) {
  c.hasQualifiedName("javax.crypto.spec", "IvParameterSpec", "IvParameterSpec") and
  i = 0
}
```

This will introduce a small code duplication when we add them as summaries in the subsequent PR, but only until this deprecated predicate gets removed.

Reply:
Yeah that makes a bit more sense. I'm fine with some minor duplication.